doc/mgr/dashboard.rst

```diff
@@ -291,42 +291,29 @@ Grafana on your preferred hosts, proceed with the following steps.
 
```

Comment:
lines 255-260 should be removed, since this PR will resolve that note.
Please also fix the link in line 272.